### PR TITLE
[yaml/fr] Fix typos

### PR DESCRIPTION
**Pull request by payet-s**

### CODE REVIEW

File: fr-fr/yaml-fr.html.markdown [hunk 1]
```diff
@@ -8,93 +8,96 @@ lang: fr-fr
...
-    Tout ce bloc de texte sera la valeur de la clé 'bloc_littéral', 
-    avec préservation des retours à la ligne. ( chaque ligne vide à 
-    l'intérieur du même bloc, sera remplacée par "\n\n" )
+    Tout ce bloc de texte sera la valeur de la clé "bloc_littéral", 
+    avec préservation des retours à la ligne.
 
     Le littéral continue jusqu'à ce que l'indentation soit annulée.
 
-        Toutes lignes qui serait "d'avantage indentées" conservent leur 
+        Toutes lignes qui seraient "d'avantage indentées" conservent leur
```
Comment (vendethiel) [inline fr-fr/yaml-fr.html.markdown]: davantage*

File: fr-fr/yaml-fr.html.markdown [hunk 2]
```diff
@@ -103,18 +106,18 @@ json_seq: [1, 2, 3, "soleil"]
...
 contenu_ancré: &nom_ancre Cette chaîne sera la valeur des deux clés.
 autre_ancre: *nom_ancre
 
-# Avec les Tags YAML, vous pouvez explicitement déclarer des types de données.
+# Avec les tags YAML, vous pouvez explicitement déclarer des types de données.
 chaine_explicite: !!str 0.5
 
-# Certains parsers implémentent des tags spécifiques à d'autres langages, 
-# comme par exemple le "complex number" de Python.
+# Certains analyseurs syntaxiques implémentent des tags spécifiques à d'autres
```
Comment (vendethiel) [inline fr-fr/yaml-fr.html.markdown]: -1

**Comment by payet-s** [comment]
@vendethiel 
Thanks for proofreading, I fixed others typos I missed such as "davantage" you stated, the introduction where "de chez" was a little weird, and "éditable" which isn't in the french dictionary even if the verb "éditer" exists.

Anyway, I don't see any good reason to keep the word "parser" that isn't even a french word while the proper way to tell it in french is "analyseur syntaxique" (i.e syntactic analyzer which is the same as parser) see the [Wikipedia page (fr)](https://fr.wikipedia.org/wiki/Analyse_syntaxique) [(en)](https://en.wikipedia.org/wiki/Parsing).

**Comment by vendethiel** [comment]
> Anyway, I don't see any good reason to keep the word "parser"

then, at least add parser in parentheses next to it, please.

**Comment by payet-s** [comment]
Done.

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**